spec/unit/forge/connection/connection_failure_spec.rb

```diff
@@ -7,7 +7,7 @@
       builder.use(:connection_failure)
 
       builder.adapter :test do |stub|
-        stub.get('/connectfail') { raise Faraday::ConnectionFailed.new(SocketError.new("getaddrinfo: Name or service not known"), :hi) }
+        stub.get('/connectfail') { raise Faraday::ConnectionFailed.new(Faraday::Adapter::Test::NetHttp::SocketError.new("getaddrinfo: Name or service not known"), :hi) }
```

Comment:
This causes a warning in tests. Let's require this class just above in order to avoid the warnings, but comment that's why we're doing it.